packages/dhis2/src/meta/helper.js

```diff
@@ -0,0 +1,72 @@
+import axios from 'axios';
+import { generateUrl } from '../Utils.js';
+
+const createHelper = (configuration = {}) => {
+  const { username, password } = configuration;
+
+  const get = url =>
+    axios({
+      url,
+      headers: { 'Content-Type': 'application/json' },
+      responseType: 'json',
+      auth: { username, password },
+      params: { paging: false },
+    });
+
+  // get resource types (static, incomplete)
+  const getResourceTypes = async () => {
+    return [
```

Comment:
This list is static and incomplete and I am not really sure HOW to complete it...